reactor-client/src/test/java/com/aerospike/client/reactor/util/Args.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2012-2018 Aerospike, Inc.
+ *
+ * Portions may be licensed to Aerospike, Inc. under one or more contributor
+ * license agreements WHICH ARE COMPATIBLE WITH THE APACHE LICENSE, VERSION 2.0.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.aerospike.client.reactor.util;
+
+import com.aerospike.client.AerospikeClient;
+import com.aerospike.client.AerospikeException;
+import com.aerospike.client.Info;
+import com.aerospike.client.async.EventLoopType;
+import com.aerospike.client.cluster.Node;
+import com.aerospike.client.policy.AuthMode;
+import com.aerospike.client.policy.TlsPolicy;
+
+import java.util.Map;
+
+public class Args {
```

Comment:
it is not a good idea to copy paste test code.
consider extracting test classes into separate module like `aerospike-client-test`

Reply:
It's not full copy.